app/controllers/application_controller.rb

```diff
@@ -2,4 +2,20 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user
+
+  def current_user
+    if session[:user_id]
+      User.find_by(id: session[:user_id])
+    else
+      false
+    end
```

Comment:
You don't need the else false - you could do User.find_by(id: session[:user_id]) if session session[:user_id] 
as a single line. If there's no such key in session, the method returns nil, which is falsy.